tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_browser_Browser.java

```diff
@@ -1043,6 +1124,7 @@ public void test_setText() {
 		assertTrue("Expecting setText() to return true", opSuccess);
 	};
 	validateTitleChanged(expectedTitle, browserSetFunc);
+	assertTrue("Browser didn't navigate to the implicit custom text location", browser.isLocationForCustomText(browser.getUrl()));
```

Comment:
This assertion seems misplaced. The test is about validating the behavior of `setText()`, now it might fail because `isLocationForCustomText()` is buggy. So if this is a relevant scenario that is not already covered by the `test_isLocationForCustomText_*()` method, please add an according one instead of mixing up concerns here.

Reply:
It has been tested for setText in many isLocationForCustomText tests. So, I'll just remove this here.